brave/browser-laptop
# Submenus are now fully navigable via keyboard.

getMenuItem now searches recursively. Needed to find nested bookmarks when
using the clickMenubarSubmenu window action.

Fixes https://github.com/brave/browser-laptop/issues/4143

Auditors: @bbondy

Test Plan:
1) Setup your bookmarks so that you have several nested folders containing bookmarks
2) Launch Brave and hit ALT to engage with the menu
3) Use arrow keys to move over to Bookmarks menu and press down
4) Navigate down to your folder and pretty right arrow to go into the folder
5) Press left arrow to "close" the submenu, returning back a level
6) Press right/left on an item which is NOT a submenu and notice it changes the menu selection (from File to Edit, for example)
